# goal
int32 distance
int32 theta
---
# result
bool flag_success
int32 final_x
int32 final_y
int32 final_theta
---
# feedback
int32 cur_x
int32 cur_y
int32 cur_theta